ecl/eclcc/eclcc.cpp

```diff
@@ -853,7 +853,8 @@ void gatherResourceManifestFilenames(EclCompileInstance & instance, StringArray
             getFullFileName(filename, true).append(".manifest");
             if (filenames.contains(filename))
                 continue;
-            if (checkFileExists(filename))
+            Owned<IFile> manifest = createIFile(filename);
+            if (manifest->exists())
```

Comment:
How about the call in  ResourceManifest::checkResourceFilesExist?
Also potentially isManifestFileValid, although much less likely.